src/mgr/ActivePyModules.cc

```diff
@@ -905,10 +972,10 @@ PyObject* ActivePyModules::get_counter_python(
       }
     }
   };
-  return with_perf_counters(extract_counters, svc_name, svc_id, path);
+  return with_unlabled_perf_counters(extract_counters, svc_name, svc_id, path);
 }
 
-PyObject* ActivePyModules::get_latest_counter_python(
+PyObject* ActivePyModules::get_latest_unlabeled_counter_python(
```

Comment:
> Can you explain why `ActivePyModules::with_unlabled_perf_counters()` and `ActivePyModules::with_perf_counters()` are separate methods/why both are needed?

Same question for `get_unlabeled_counter_python()` and `get_latest_unlabeled_counter_python()` -- can we get rid of all this cut-and-paste?

Reply:
similar reasoning as https://github.com/ceph/ceph/pull/61997#discussion_r1992815786